tests/integration_tests/functional/test_drive_vhost_user.py

```diff
@@ -60,3 +113,182 @@ def test_vhost_user_block(microvm_factory, guest_kernel, rootfs_ubuntu_22):
     # Verify if guest can run commands.
     exit_code, _, _ = vm.ssh.run("ls")
     assert exit_code == 0
+
+    # Prepare the input for doing the assertion
+    assert_dict = {}
+    # Keep an array of strings specifying the location where some string
+    # from the output is located.
+    # 1-0 means line 1, column 0.
+    keys_array = ["1-0", "1-6"]
+    # Keep a dictionary where the keys are the location and the values
+    # represent the input to assert against.
+    assert_dict[keys_array[0]] = "ro"
```

Comment:
can we add an explicit test that both `ro` and `rw` cases are respected by Firecracker?

Reply:
yes, my point is we can configure the backend both ways and see if FC gives proper indication about it to the guest.

Reply:
oh, I totally forgot I configured backend to always be readonly. I will add a test with `rw`.

Reply:
added a test with `rw` block